doc/developers/develop.rst

```diff
@@ -410,6 +410,27 @@ inside the** ``__init__`` **method**. All and only the public attributes set by
 fit have a trailing ``_``. As a result the existence of parameters with
 trailing ``_`` is used to check if the estimator has been fitted.
 
+Links to the documentation
+--------------------------
+Estimators include a method ``_get_url_link``, which is used to get the URL
+of the documentation of the estimator. This function is used to prepare the html
+representation of the estimator, e.g., when using the ``display`` function in a
+Jupyter notebook.
+
+By default, the function returns the empty string if the estimator is not a
+scikit-learn estimator. This check is performed by evaluating whether the
```

Comment:
```suggestion
By default, the function returns the empty string if the estimator is not a
part of the scikit-learn library. This check is performed by evaluating whether the
```
I had to think about this a bit before I realise what you meant. Not sure if my suggestion is an improvement or not :-/

Basically the first time I read the paragraph "scikit-learn estimator" included third party, scikit-learn compatible estimators for me. But then the paragraph makes no sense. Which is why I had to read it twice.